server/views/forms/summary-new.njk

```diff
@@ -29,9 +31,14 @@
         }) }}
       </div>
     </div>
+    {% endif %}
```

Comment:
Nice, although I think this `endif` needs to be moved up a line as it appears to enclose one too many closing `div` tags

Reply:
Done!